specs/table-schema.md

```diff
@@ -410,7 +410,7 @@ properties.
       All
     </td>
     <td>
-      Indicates whether this field is allowed to be `null`. If required is `true`, then `null` is disallowed. See the section on `missingValues` for how, in the physical representation of the data, strings can represent `null` values.
```

Comment:
I get the logic of this change though I do find the new version a little less readable in the second sentence (tho' prob clearer in the first).